Updated readme to include neovim snippet (#108)
bram209/leptosfmt: README.md

```json
  "rust-analyzer.rustfmt.overrideCommand": ["leptosfmt", "--stdin", "--rustfmt"]
```
> Note: For VSCode users, I recommend to use workpsace settings (CMD + shift + p -> Open workspace settings), so that you can only configure `leptosfmt` for workpsaces that are using leptos. For Neovim users,

> Alternatively, if you are using [nvim-lspconfg](https://github.com/neovim/nvim-lspconfig), you may append the following to your `.setup{}` table:

Reviewer: I am pretty sure that `neoconf.nvim` still utilises `nvim-lspconfig` under the hood.
```suggestion
Alternatively, you can configure [nvim-lspconfig](https://github.com/neovim/nvim-lspconfig) directly, you may append the following to your `.setup{}` table:
```

Reviewer: we could also point them to the docs of `nvim-lspconfig`: https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#rust_analyzer

Author: roger, I'll make these changes
